scripts/travis/script_deploy_android_snapshot.sh

```diff
@@ -8,16 +8,8 @@ if [ "${PLATFORM}" = "android" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [
     # Configure private repository credentials (used to sign release artifacts)
     echo -e "machine github.com\n  login $GITHUB_USERNAME\n  password $GITHUB_PASSWORD" >> ~/.netrc
 
-    # Build all android architectures (armeabi-v7a already build)
-    make android-native-lib ANDROID_ARCH=x86
-    make android-native-lib ANDROID_ARCH=armeabi
-    make android-native-lib ANDROID_ARCH=arm64-v8a
-    #### Currently Not building the following architectures
-    #make android-native-lib ANDROID_ARCH=x86_64
-    #make android-native-lib ANDROID_ARCH=mips
-    #make android-native-lib ANDROID_ARCH=mips64
+    make android-sdk
```

Comment:
So previously `arm7l` was done during the default build process and other architecture were build only during deployment.

It seems this will build, everything again including arm7l (in other words, arm7l will be builds twice).